docs/rules/one-var.md

```diff
@@ -46,6 +63,15 @@ The following patterns are considered warnings:
 function foo() {
```

Comment:
Thanks for the catch. I did the second to show that `const` is also affected. Maybe I'll just lump `const` into the first example and screw the foobar terminology?